x/ecocredit/server/testsuite/suite.go

```diff
@@ -973,6 +973,14 @@ func (s *IntegrationTestSuite) TestScenario() {
 		})
 	}
 
+	// TODO: test create sell order
+
+	// TODO: test update sell order
+
+	// TODO: test create buy order
+
+	// TODO: test allow ask denom
```

Comment:
will this be another PR or will this PR update?

Reply:
https://github.com/regen-network/regen-ledger/issues/627